cmd/get.go

```diff
@@ -63,6 +65,12 @@ func getCMDsFlags(getCmd *cobra.Command) {
 	getCmd.PersistentFlags().Bool("disable-ipv6", false, "Disable IPv6 for requests.")
 	getCmd.PersistentFlags().Bool("ipv6-anyip", false, "Use AnyIP kernel feature for requests. (only IPv6, need --random-local-ip)")
 
+	// Rate limiting flags
+	getCmd.PersistentFlags().Bool("disable-rate-limit", false, "Disable the Token Bucket rate limiting.")
+	getCmd.PersistentFlags().Float64("rate-limit-capacity", 10, "Bucket capacity for each host.")
+	getCmd.PersistentFlags().Float64("rate-limit-refill-rate", 2, "Ideal requests per second for each host.")
```

Comment:
I have no idea on chosing a sweet --rate-limit-refill-rate default value.

Reply:
I think 50 is a good default. Most hosts won't mind the rate and when there is a problem it will reduce!